nncf/quantization/metrics.py

```diff
@@ -4,6 +4,7 @@
 from texttable import Texttable
 from collections import deque
 
+# pylint:disable=unused-import
```

Comment:
Have you tried to delete `get_all_modules_by_type`? Looks like it's better to remove it this import?

Reply:
Same here, I decided not delete unused import - from nncf.utils import get_all_modules_by_type on 9 line

Reply:
It shouldn't impact the functionality. But still it worth mentioning the author

Reply:
Done